extension/surveys/httpwithans.js

```diff
@@ -0,0 +1,18 @@
+/**
```

Comment:
Yes, both the question and the screenshot will be set to real things in a follow-up PR. I'm waiting on agreement from Rob as to what the questions ought to be.